Documentation/op-guide/recovery.md

```diff
@@ -61,3 +61,9 @@ $ etcd \
 ```
 
 Now the restored etcd cluster should be available and serving the keyspace given by the snapshot.
+
+## Restoring a cluster from membership mis-reconfiguration with wrong URLs
```

Comment:
This section should be made more generic: restoring a cluster from membership misconfiguration. also the content should focus on how to restore the mis-configured cluster i believe.